tests/Unleash.Tests/Utilities/ToggleBootstrapFileProvider_Tests.cs

```diff
@@ -49,8 +43,11 @@ public void Returns_File_Content_When_File_Exists()
             var result = toggleFileProvider.Read();
 
             // Assert
-            result.Features.Count().Should().Be(3);
-            result.Features.Single(f => f.Name == "featureY").Enabled.Should().Be(false);
+            result.Should().Be(fileContent);
+
+            var deserializedResult = Newtonsoft.Json.JsonConvert.DeserializeObject<ToggleCollection>(result);
+            deserializedResult?.Features.Count().Should().Be(3);
+            deserializedResult?.Features.Single(f => f.Name == "featureY").Enabled.Should().Be(false);
```

Comment:
Guess that goes for this one too, and the other test further down?

Reply:
What are these comments in reference to? 😄

Reply:
Oh you mean null assertion on the `deserializedResult`?

Reply:
Let me know if this is better: https://github.com/Unleash/unleash-client-dotnet/pull/224/commits/fe0f1cc65634e24d06fbaaaa1b14cd3e402aeaf8

Reply:
Apologies for lack of context there, but yes! and thanks!